charts/spire/README.md

```diff
@@ -1,8 +1,6 @@
 # spire
 
-<!-- This README.md is generated. -->
```

Comment:
Can you use the script from the .github folder to generate the docs?

So the template is used.